src/utils/py/daos_api.py

```diff
@@ -1639,6 +1639,22 @@ def commit_tx(self, txn):
             raise DaosApiError("TX commit returned non-zero. RC: {0}"
                                .format(ret))
 
+    def close_tx(self, txn):
+        """ close out a transaction that is done being modified """
```

Comment:
Could you please provide some more informative description to this function. It looks exactly similar to the one above for commit_tx(). It could be confusing. If commit_tx() description's needs to be modified as well to have some distinction between these two, please go ahead and do so.
Thanks.

Reply:
Under  class DaosContext,  action-subject  self.ftable = { .. }
open-tx, commit-tx and close-tx has similliar method calls to self.context.get_function
close-tx was missing or unimplemented.
should not be an issue.